config/src/test/java/org/springframework/security/config/annotation/method/configuration/MethodSecurityServiceConfig.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2002-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.security.config.annotation.method.configuration;
+
+import org.springframework.context.annotation.Bean;
+
+/**
+ * @author Josh Cummings
+ */
+public interface MethodSecurityServiceConfig {
```

Comment:
If you are going to extract this out, I'd do so as a class for consistency with the other tests. Then in the tests just register multiple configurations.

Reply:
great, will do